.appveyor.yml

```diff
@@ -0,0 +1,133 @@
+version: '{branch}-{build}'
+os: Visual Studio 2015
+clone_folder: C:\projects\oiio
+test: off
+configuration:
+    - Release
+    - Debug
+platform:
+    - x64
+# branches:
+#   only:
+#     - master
+environment:
+  matrix:
+    # - CMAKE_PLATFORM: "Visual Studio 14 2015"
+    #   PYTHON_DIR: "C:\\Python27"
+    - CMAKE_PLATFORM: "Visual Studio 14 2015 Win64"
+      PYTHON_DIR: "C:\\Python27-x64"
+      tbs_arch: "x64"
+      tbs_tools: "msvc14"
+      tbs_static_runtime: 0
+
+install:
+  - cinstall: python
+  - cinst ninja
```

Comment:
The distinction is that I probably cut one line from one project's appveyor.yml, and the other line from a different project's appveyor.yml.

This is a truly Frankenstein's monster of build scripts. I literally don't know what 90% does.

I'm flying blind here, googling other projects' appveyor files and cutting and pasting the bits that look like they do things that my project needs. :-)